package.json

```diff
@@ -4,6 +4,9 @@
   "author": "Takuya Asano <takuya.a@gmail.com>",
   "description": "JavaScript implementation of Japanese morphological analyzer",
   "main": "./dist/node/kuromoji.js",
+  "browser": {
```

Comment:
[browser field spec for package.json](https://gist.github.com/defunctzombie/4339901 "browser field spec for package.json")